projects/core/src/routing/configurable-routes/config/default-storefront-routes-translations.ts

```diff
@@ -0,0 +1,55 @@
+import { StorefrontRoutesTranslations } from './storefront-routes-translations';
+
+export const defaultStorefrontRoutesTranslations: {
+  default?: StorefrontRoutesTranslations;
+  [languageCode: string]: StorefrontRoutesTranslations;
+} = {
+  default: {
+    homepage: { paths: [''] },
+    cart: { paths: ['cart'] },
+    search: { paths: ['search/:query'] },
+    login: { paths: ['login'] },
+    register: { paths: ['register'] },
+    resetNewPassword: { paths: ['reset-new-password/:token'] },
+    resetPassword: { paths: ['reset-password'] },
+    checkout: { paths: ['checkout'] },
+    orderConfirmation: { paths: ['order-confirmation'] },
+    product: {
+      paths: ['product/:productCode'],
+      paramsMapping: { productCode: 'code' }
+    },
+    category: {
+      paths: [
+        'category/:categoryCode/:title',
+        'category/:categoryCode',
+        'Brands/:brandName/c/:brandCode'
+      ],
+      paramsMapping: { categoryCode: 'code' }
+    },
+    storeFinder: {
+      paths: ['store-finder'],
+      children: {
+        searchResults: { paths: ['find-stores'] },
+        allStores: { paths: ['view-all-stores'] },
+        listStores: {
+          paths: ['country/:country/region/:region', 'country/:country']
+        },
+        storeDescription: {
+          paths: ['country/:country/region/:region/:store']
+        }
+      }
+    },
+    termsAndConditions: { paths: ['terms-and-conditions'] },
```

Comment:
as discussed, the following are CMS driven and should not be part of the hard-coded config:
- termsAndConditions
- contact
- help
- sale

Reply:
I'll put a note in documentation.
And as discussed offline, I removed `myAccount_` prefix from config keys